workbench-core/example/app/src/routes/datasetRoutes.ts

```diff
@@ -98,7 +117,7 @@ export function setUpDSRoutes(
   router.delete(
     '/datasets/:id',
     wrapAsync(async (req: Request, res: Response) => {
-      if (req.params.id.match(uuidWithLowercasePrefixRegExp(resourceTypeToKey.dataset)) === null) {
+      if (req.params.id.match(uuidWithLowercasePrefixRegExp(dataSetPrefix)) === null) {
         throw Boom.badRequest('id request parameter is invalid');
       }
       await dataSetService.removeDataSet(req.params.id);
```

Comment:
should we catch the `DataSetHasEndpointError` here and throw a Boom error so we avoid a 5xx error code?

Reply:
I saw that in your code. I will circle back and fix that.